numpy/core/__init__.py

```diff
@@ -16,10 +16,31 @@
     from . import multiarray
 except ImportError as exc:
     msg = """
+
+IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!
+
 Importing the multiarray numpy extension module failed.  Most
 likely you are trying to import a failed build of numpy.
-If you're working with a numpy git repo, try `git clean -xdf` (removes all
-files not under version control).  Otherwise reinstall numpy.
+Here is how to proceed:
+- If you're working with a numpy git repository, try `git clean -xdf`
+  (removes all files not under version control) and rebuild numpy.
+- If you are simply trying to use the numpy version that you have installed:
+  your installation is broken - please reinstall numpy.
+- If you have already reinstalled and that did not fix the problem, then:
+  1. Check that you are using the Python you expect (e.g. check the path with
+     `import os; print(os.__file__)`), and that you have no directories in
+     your PATH or PYTHONPATH that can interfere with the Python and numpy
+     versions you're trying to use.
+  2. If (1) looks fine, you can open a new issue at
+     https://github.com/numpy/numpy/issues.  Please include details on:
```

Comment:
We could maybe use an [issue template](https://help.github.com/articles/creating-an-issue-template-for-your-repository/) at `.github/ISSUE_TEMPLATE/install_failed.md`, and then just link to a page that uses that template.

Reply:
I'd like to keep the plain text as well, but an issue template could be helpful in this case. Default issue templates are usually less than helpful (for most projects I just end up deleting them because they don't cover whatever case I'm reporting well), but here we have a pretty constrained problem.

Reply:
What do others think, is it worth adding `.github/ISSUE_TEMPLATE/install_failed.md` and link to it?

Reply:
I would think the first stage is to commit this PR, then if we add a template think about linking to it. Links are problematic, many NumPy users are not connected to the internet.

Reply:
> We could maybe use an issue template at .github/ISSUE_TEMPLATE/install_failed.md, and then just link to a page that uses that template.

Point 1 in the new text is required for non-connected users. I wanted to make sure we do not delete that text in favor of an on-line solution.